src/app/screens/services/EmploiDuTemps.tsx

```
@@ -45,79 +203,151 @@ export const EmploiDuTemps = () => {
+          {/* cours */}
+          {filteredCourses.map((cours, i) => {
+            const startInMin = toMinutes(cours.heure_debut);
+            const endInMin = toMinutes(cours.heure_fin);
+            const baseInMin = START_HOUR * 60 - 14; // pour décalage top, pour synchro sur les heures
+            const top = ((startInMin - baseInMin) / 60) * HOUR_HEIGHT;
+            const height = ((endInMin - startInMin) / 60) * HOUR_HEIGHT;
+
+            return (
+              <View
+                key={i}
+                style={{ top, height }}
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)